microsoft-edge/webview2/release-notes/index.md

```diff
@@ -63,6 +63,305 @@ No additional APIs have been promoted to Stable and added in this Release SDK.
 <!-- end of Jan 2025 Release SDK -->
 
 
+<!-- ====================================================================== -->
+## 1.0.3079-prerelease
+
+Release Date: January 23, 2025
+
+[NuGet package for WebView2 SDK 1.0.3079-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3079-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 134.0.3079.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+The following Experimental APIs have been added in this Prerelease SDK.
+
+
+<!-- ---------- -->
+###### DefaultBackgroundColor API<!-- todo: feature-oriented heading -->
+
+<!-- todo: lead-in - https://github.com/MicrosoftEdge/WebView2Feedback/pull/4981/files -->
```

Comment:
create lead-in re: default bg color; see https://github.com/MicrosoftEdge/WebView2Feedback/pull/4981/files